measures/SimulationOutputReport/measure.rb

```diff
@@ -37,8 +37,8 @@ def arguments
 
     # make an argument for including optional end use subcategories
     arg = OpenStudio::Measure::OSArgument::makeBoolArgument("include_enduse_subcategories", true)
-    arg.setDisplayName("Include End Use Subcategories")
-    arg.setDescription("Whether to report end use subcategories: appliances, plug loads, fans, large uncommon loads.")
+    arg.setDisplayName("Include Disaggregated Interior Equipment")
+    arg.setDescription("Whether to report interior equipment broken out into components: appliances, plug loads, exhaust fans, large uncommon loads, etc. For each fuel type, total/net energy consumption will not be reported.")
     arg.setDefaultValue(false)
```

Comment:
Can we default this to True?

Reply:
Yes, although I think this would then require change to buildstockbatch, right?